changelog.d/pa-2263.fixed

```diff
@@ -0,0 +1 @@
+DeepSemgrep: Semgrep no longer crashes in certain cases when `metavariable-pattern/regex` are used on a metavariable referencing another metavariable.
```

Comment:
Perhaps I got it wrong, but I think this changelog entry is not accurate. The error happens when using those operators on a metavariable _that binds to a code variable_. And Semgrep never crashed AFAICT. It's printed like "internal error" but it's an error we are recovering from. It should probably just say "matching warning" or something. The thing is that the operators now should work, and previously the just evaluated to false.